dev/bots/run_command.dart

```diff
@@ -188,23 +188,15 @@ Future<CommandResult> runCommand(String executable, List<String> arguments, {
         print(result.flattenedStdout);
         print(result.flattenedStderr);
     }
-    String allOutput;
-    if (failureMessage == null) {
-      allOutput = '${result.flattenedStdout}\n${result.flattenedStderr}';
-      if (allOutput.split('\n').length > 10) {
-        allOutput = '(stdout/stderr output was more than 10 lines)';
-      }
-    } else {
-      allOutput = '';
-    }
+    final String allOutput = '${result.flattenedStdout}\n${result.flattenedStderr}';
     foundError(<String>[
       if (failureMessage != null)
         failureMessage,
       '${bold}Command: $green$commandDescription$reset',
       if (failureMessage == null)
         '$bold${red}Command exited with exit code ${result.exitCode} but expected ${expectNonZeroExit ? (expectedExitCode ?? 'non-zero') : 'zero'} exit code.$reset',
       '${bold}Working directory: $cyan${path.absolute(relativeWorkingDir)}$reset',
-      if (allOutput.isNotEmpty)
+      if (allOutput.isNotEmpty && allOutput.length < 10)
```

Comment:
This omits output if it is longer than 10 _characters_? That seems a bit short? (used to be 10 _lines_) (we really should have a test for this...)

Also, should we continue to log that output was omitted because it was too long?

Reply:
woops yeah the 10 characters thing was definitely a mistake. i'll see if it's testable.

i removed the log saying it was too long because it's just kind of superfluous. in practice it doesn't really help anything as far as i can tell. it's not like the reader is looking for the output, they'll just go find the original logs anyway.